plugwise/__init__.py

```diff
@@ -318,6 +338,7 @@ def __init__(
         SmileData.__init__(self)
 
         self.smile_hostname: str | None = None
+        self._previous: str = "0"
```

Comment:
I'd suggest better naming to prevent future confusion (also with the 'new' in the update routine)

Reply:
Yeah, hadn't come to that, done :)
And it helped me to see a bug.